src/Symfony/Component/Validator/Tests/Constraints/PasswordStrengthValidatorWithClosureTest.php

```diff
@@ -0,0 +1,98 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Validator\Tests\Constraints;
+
+use Symfony\Component\Validator\Constraints\PasswordStrength;
+use Symfony\Component\Validator\Constraints\PasswordStrengthValidator;
+use Symfony\Component\Validator\Test\ConstraintValidatorTestCase;
+use Symfony\Component\Validator\Tests\Constraints\Fixtures\StringableValue;
+
+/**
+ * @group legacy
+ */
+class PasswordStrengthValidatorWithClosureTest extends ConstraintValidatorTestCase
+{
+    protected function createValidator(): PasswordStrengthValidator
+    {
+        return new PasswordStrengthValidator(PasswordStrengthValidator::estimateStrength(...));
```

Comment:
Shouldn't this be a _different_ closure than the default one? Otherwise we don't really test if the passed closure is actually called.

Reply:
indeed, changed for a simple closure based on the length just for the test